evap/results/tests/test_views.py

```diff
@@ -73,6 +80,27 @@ def test_heading_question_filtering(self):
         self.assertIn(likert_question.text, page)
         self.assertNotIn(heading_question_2.text, page)
 
+    def test_many_answers_course_no_warning(self):
+        self._create_questionaire_with_answers(answer_count=10)
+        url = '/results/semester/%s/course/%s' % (self.semester.id, self.course.id)
+        user = 'evap'
+        page = self.get_assert_200(url, user)
+        self.assertNotIn("Only a few participants answered these questions.", page)
+
+    def test_zero_answers_course_no_warning(self):
+        self._create_questionaire_with_answers(answer_count=0)
+        url = '/results/semester/%s/course/%s' % (self.semester.id, self.course.id)
+        user = 'evap'
+        page = self.get_assert_200(url, user)
+        self.assertNotIn("Only a few participants answered these questions.", page)
+
+    def test_few_answers_course_show_warning(self):
+        self._create_questionaire_with_answers(answer_count=3)
+        url = '/results/semester/%s/course/%s' % (self.semester.id, self.course.id)
+        user = 'evap'
+        page = self.get_assert_200(url, user)
```

Comment:
you can put `user='evap'` into the parameter list directly, no need to introduce a variable for that.